app/plugin.go

```diff
@@ -567,3 +622,33 @@ func (a *App) notifyPluginEnabled(manifest *model.Manifest) error {
 
 	return nil
 }
+
+func (a *App) writeFile(filename string, key io.Reader) *model.AppError {
```

Comment:
This method is quite generic and can be used all over the `app`. It just saves the file to the filestore.

Reply:
I believe it's writing to the configStore as opposed to the regular FileStore, so this operation isn't that common. I feel like this API should be on the FS interface rather than us creating a wrapper for it. My personal recommendation is: since this is the 2nd time we're calling this function across our server repo, I say we just do this inline rather than creating a helper function. Later we can update `SetFile(name string, data []byte) error` to  `SetFile(name string, reader io.Reader) error`. 2/5 on this though. If you do feel strongly about this then I would suggest to move this into some sort of common/utils location so others can use it since it'll be weird to have a common helper function that's sitting in this file.
Thoughts?